custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/score.R

```diff
@@ -135,6 +135,33 @@ load_serialized_model <- function(model_dir, target_type) {
     data
 }
 
+.match_float_labels <- function(class_labels, predict_labels) {
```

Comment:
how come we can't do this in the python?

Reply:
Because `predict_labels` doesn't exist in Python.

Reply:
I thought we returned it...

Reply:
we do return predictions from R, but that's after the labels are validated so it's too late.